### PR TITLE
docs(setup): Halyard commands to set up Slack notifications

### PR DESCRIPTION
**Pull request by dorbin**
#377

### CODE REVIEW

File: setup/features/notifications/index.md [hunk 1]
```diff
@@ -102,26 +102,13 @@ Note: your users will need to invite the hipchat bot to private rooms that want
...
-  notifications: {
-    slack: {
-      enabled: true,
-      botName: '<username of bot>'
-    },
-// ...
-```
+and
+
+`hal config notification slack edit --bot-name $SPINNAKER_BOT --token $TOKEN_FROM_SLACK`
```
Comment (lwander) [inline setup/features/notifications/index.md]: I would format this in one large block with 

\`\`\`bash
hal config notification slack enable
hal config notification slack edit --bot-name $SPINNAKER_BOT --token $TOKEN_FROM_SLACK
\`\`\`

Comment (lwander) [reply setup/features/notifications/index.md]: This adds bash syntax highlighting & groups the commands together

File: setup/features/notifications/index.md [hunk 2]
```diff
@@ -102,25 +102,11 @@ Note: your users will need to invite the hipchat bot to private rooms that want
...
-// ...
-  notifications: {
-    slack: {
-      enabled: true,
-      botName: '<username of bot>'
-    },
-// ...
+```bash
+hal config notification slack enable`
+`hal config notification slack edit --bot-name $SPINNAKER_BOT --token $TOKEN_FROM_SLACK
```
Comment (lwander) [inline setup/features/notifications/index.md]: You can drop the \` here and on the line above

Comment (dorbin) [reply setup/features/notifications/index.md]: haha, careless...

**Review by lwander — APPROVED** [review]

**Pull request merged**